infra/metadata.display.yaml

```diff
@@ -16,8 +16,6 @@ apiVersion: blueprints.cloud.google.com/v1alpha1
 kind: BlueprintMetadata
 metadata:
   name: terraform-ml-image-annotation-gcf-display
-  annotations:
```

Comment:
I'm not sure what this does.

Reply:
All changes to the metadata.yaml and metadata.display.yaml were auto-generated when I ran `make docker_generate_docs`. ATM these files are not used anywhere. But we are working with the blueprints team to improve the process. So these changes can be ignored.